models/links.go

```diff
@@ -0,0 +1,42 @@
+package models
+
+import (
+	"net/url"
+)
+
+const (
+	codeListURI   = "/code-lists/%s"
+	editionsURI   = "/code-lists/%s/editions"
+	editionURI    = "/code-lists/%s/editions/%s"
+	codesURI      = "/code-lists/%s/editions/%s/codes"
+	codeURI       = "/code-lists/%s/editions/%s/codes/%s"
+	datasetsURI   = "/code-lists/%s/editions/%s/codes/%s/datasets"
+	datasetAPIuri = "/datasets/%s"
+)
+
+// Link contains the id and a link to a resource
+type Link struct {
+	ID   string `json:"id,omitempty"     bson:"id"`
+	Href string `json:"href"             bson:"href"`
+}
+
+func CreateLink(id, href, host string) *Link {
+
+	rel, err := url.Parse(href)
+	if err != nil {
```

Comment:
Should this func return an error? If not consider logging the error here before returning nil. Helpful to see that it has happened even if we don't return it.

Reply:
Yes, it makes sense. As this is a `models` packages, I think it would make more sense to return the error, so that when we call the method, we can decide what to do with it.